### PR TITLE
StackItem, Unsafe and ExecutionContext 100% coverage

### PR DESCRIPTION
**Pull request by shargon**

### CODE REVIEW

File: tests/neo-vm.Tests/UtStackItem.cs [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+            Assert.IsInstanceOfType(item, typeof(ByteArray));
+            CollectionAssert.AreEqual(new byte[] { 0x01, 0x02, 0x03 }, item.GetByteArray());
+
+            // String
+
+            item = "NEO";
+
+            Assert.IsInstanceOfType(item, typeof(ByteArray));
+            CollectionAssert.AreEqual(Encoding.UTF8.GetBytes("NEO"), item.GetByteArray());
+            Assert.AreEqual("NEO", item.GetString());
```
Comment (igormcoelho) [inline tests/neo-vm.Tests/UtStackItem.cs]: utf8 it would be nice to have non-ascii char. Any chinese test? :)

Comment (shargon) [reply tests/neo-vm.Tests/UtStackItem.cs]: Agree. Any idea :) ?

File: src/neo-vm/StackItem.cs [hunk 2]
```diff
@@ -10,8 +10,6 @@ namespace Neo.VM
 {
     public abstract class StackItem : IEquatable<StackItem>
     {
-        public static StackItem Null { get; } = new byte[0];
```
Comment (erikzhang) [inline src/neo-vm/StackItem.cs]: Why remove this? The public fields may be used in other projects.

File: tests/neo-vm.Tests/UtStackItem.cs (hunk 1 above)
Comment (shargon) [reply tests/neo-vm.Tests/UtStackItem.cs]: Done

**Comment by shargon** [comment]
Ready to merge

**Pull request merged**